patterns/publishAndSuscribe.js

```diff
@@ -0,0 +1,96 @@
+/*
+ PUBLISH AND SUSCRIBE
+
+ This pattern is a case of Observer pattern. The difference is that the observers subscribe to
+ a particular event from an event manager and the subject can fire events that make the manager notify
+ to the observers.
+
+*/
+
+// Subject
+class BoardGameTable {
+  constructor(maxPlayers, gameName) {
+    this.spaceInTable = maxPlayers;
+    this.game = gameName;
+  }
```

Comment:
suggestion: maybe we can use a publishing house as an example with journals and newspapers being sent to readers?